datasets/es/mayors_councillors/es_mayors_councillors.yml

```diff
@@ -0,0 +1,86 @@
+title: Spain Mayors and Councillors
+entry_point: crawler.py
+prefix: es-mc
+load_statements: true
+coverage:
+  frequency: daily
+  start: 2025-07-24
+summary: >
+  Individuals holding mayoral or councillor positions in Spain.
+description: |
+  This dataset contains detailed information on individuals holding mayoral or councillor 
+  positions in Spain. Each entry includes the name of the official, their position, and 
+  political affiliation. The data covers current officials and historical data from the 
+  2019-2023 term.
+publisher:
+  name: Ministerio de Política Territorial y Memoria Democrática
+  name_en: Ministry of Territorial Policy and Democratic Memory
+  acronym: MPTMD
+  official: true
+  description: |
+    The Ministry of Territorial Policy and Democratic Memory is the government department 
+    responsible for proposing and executing policy on relations and cooperation with 
+    autonomous communities and local administrations.
+  country: es
+  url: https://mpt.gob.es/en/ministerio.html
+tags:
+  - list.pep
+url: https://mpt.gob.es/en/politica-territorial/local/sistema_de_informacion_local_-SIL-/alcaldes_y_concejales.html
+data:
+  url: https://concejales.redsara.es/consulta/
```

Comment:
have you seen any suggestion that this data gets updated with by-elections or is it just the original election result? I've spent way too much time trying to figure this but it's tricky.

Reply:
Ignore what I said before, I had a different version, the list is indeed updated and mantained. Yes, the dates for the replacements of the councillors that resign are the correct dates.